src/sentry/integrations/slack/message_builder/base/block.py

```diff
@@ -176,7 +176,7 @@ def _build_blocks(
         *args: SlackBlock,
         fallback_text: str | None = None,
         color: str | None = None,
-        block_id: dict[str, int] | None = None,
+        block_id: dict[str, int] | str | None = None,
```

Comment:
This is a bit of an unusual type. Are we calling this with both dictionaries and strings?

Reply:
i'm not entirely sure, i can try to remove it and see if anything fails. we're inserting whatever `block_id` is into a dict though

Reply:
ok nothing fails, we're good to go